.circleci/config.yml

```diff
@@ -49,15 +60,22 @@ jobs:
   npm_publish:
     working_directory: ~/aerogear
     docker:
-      # Node 8 LTS
       - image: circleci/node:lts
     steps:
       - checkout
-        # Allows us to authenticate with the npm registry
-      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
-      - run: CI=true npm run release:prep
-      - run: TAG=$CIRCLE_TAG npm run release:validate
-      - run: TAG=$CIRCLE_TAG npm run publish
+      - publish:
+          tag: "$CIRCLE_TAG"
+  npm_publish_master:
+    working_directory: ~/aerogear
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout
+      - run:
+          name: version packages
+          command: npx lerna version prerelease --no-push --yes --preid "master.$(git rev-list HEAD --count).$(git rev-parse --short HEAD)"
```

Comment:
I would really want to have script that does this stuff so we can try this before merge on local machine and have way to replicate this changes across repositories (voyager, js and others)